issue-005/solution.js

```diff
@@ -1,4 +1,10 @@
-// implement your solution here
-function hello() {
-
-}
\ No newline at end of file
+function hello(person) {
+    switch (person) {
```

Comment:
`switch` works here but was it really the best choice for this kind of problem? Since it was binary, so `person` was either defined or `undefined`, so `if-else` would have been better here.

`switch` is best for multiple conditions.